src/advanced/swap.js

```diff
@@ -2,5 +2,8 @@ let a = 8
 let b = 10
 
 // TODO: Swap the values of a and b without changing lines 1 and 2; extra points if you can do it without using a temporary variable
+const temp = a
```

Comment:
Try and doi this without the temp variable

Reply:
I tried to do [a, b] = [b, a],
but prettier/eslint is freaktion out with that, not sure why.